actionpack/test/dispatch/request/json_params_parsing_test.rb

```diff
@@ -37,13 +37,6 @@ def teardown
     )
   end
 
-  test "does not parse unregistered media types such as application/vnd.api+json" do
-    assert_parses(
-      {},
-      "{\"person\": {\"name\": \"David\"}}", { 'CONTENT_TYPE' => 'application/vnd.api+json' }
-    )
-  end
-
```

Comment:
Well the point of this PR is to actually register the `application/vnd.api+json` media type, so I'm not sure this test is consistent with this PR.